samples/js/fb/slurp/src/main.js

```diff
@@ -62,6 +69,30 @@ main().catch(ex => {
 });
 
 async function main(): Promise<void> {
+  if (args['exchange-token']) {
+    const resp = await callFacebookRaw('/oauth/access_token?' +
+      'grant_type=fb_exchange_token&' +
+      'client_id=1025558197466738&' +
+      'client_secret=45088a81dfea0faff8f91bbc6dde0a0c&' +
+      'fb_exchange_token=' + args['access-token']);
+    const body = await resp.text();
+    if (resp.status !== 200) {
+      throw `Error ${resp.status} ${resp.statusText}: ${body}`;
+    }
+    const t = body
+      .split('&')
+      .map(kv => kv.split('='))
+      .filter(([k]) => k === 'access_token')
+      .map(([, v]) => v)
+      .shift();
+    console.log('Long-lived access token: ' + t);
```

Comment:
No, not really, because you need to be able to store the long-lived token somewhere.